extension-manifest-v3/src/background/healthcheck.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { captureException } from '/utils/errors.js';
+
+async function healthcheck() {
```

Comment:
Can we delay this a bit? Ideally, we should avoid to push non-essential IO not on the startup, because it makes it harder for other modules to initialize in time.